tests/fixtures/large_bitcoin_signature_request.json

```diff
@@ -1,306 +0,0 @@
-{
```

Comment:
why is this deleted in full? I forget why it's here.

is there a corresponding png to delete?

Reply:
The naming was confusing.  I've renamed this fixture to `too_large_bitcoin_signature_request.json` which makes it's usage more clear.  Since it's too large, there is no corresponding image.